extension/war/src/main/webapp/groovy/social/webui/UIUserInvitation.gtmpl

```diff
@@ -27,10 +28,10 @@ placeholder = placeholder?.replace("'", "\\'").replace("&#39;", "\\&#39;");
 
   <% uiform.begin();%>
   <div class="uiGrayLightBox">
-		<div class="pull-right"><button class="btn" type="button" onclick="$inviteAction">Invite</button></div>
-		<div class="mention-input">
-			<% uiform.renderField("user"); %>
-		</div>
+        <div class="mention-input">
```

Comment:
@Jihed525 can you please rectify indentation here ?

Reply:
ok